corehq/apps/reports/views.py

```diff
@@ -1304,6 +1305,31 @@ def page_context(self):
         }
 
 
+def form_to_json(domain, form, timezone=None):
+    form_name = xmlns_to_name(
+        domain,
+        form.xmlns,
+        app_id=form.app_id,
+        lang=get_language(),
+    )
+    if timezone is None:
+        received_on = json_format_datetime(form.received_on)
+    else:
+        offset = timezone.utcoffset(datetime.utcnow())
+        local_received_on = form.received_on + offset
```

Comment:
Could do `ServerTime(form.received_on).user_time(timezone)`

Reply:
Oh, didn't know about this. Will update.